Bottom: b32e7bc1ac2111ad1bac924b1876488fa8206f72
Top:    51dc2247accf28c71558fc7f6b37c2b4bc854ec8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 14:13:31 +0000

Disable polygon heritage


---

diff --git a/src/polygon1.c b/src/polygon1.c
index b652377..7b84cf4 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2842,8 +2842,10 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   *res = NULL;
 
   /* Make copies for tracking polygon parentage (DEBUG) */
-  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
-      return err_no_memory;
+//  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
+//      return err_no_memory;
+  a_copy = NULL;
+  b_copy = NULL;
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
